README.md

```diff
@@ -26,41 +26,48 @@ We support the two major Go versions, which are 1.14 and 1.15 at the moment.
 # Examples 
 
 ## Initiate sign request
+
 ```go
+package main
+
 import (
-    "context"
-    "fmt"
-    "io/ioutil"
-    "github.com/NicklasWallgren/bankid"
-    "github.com/NicklasWallgren/bankid/configuration"
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/NicklasWallgren/bankid"
+	"github.com/NicklasWallgren/bankid/configuration"
 )
 
-certificate, err := ioutil.ReadFile("path/to/environment.p12")
-if err != nil {
-    panic(err)
-}
+func main() {
+	certificate, err := ioutil.ReadFile("path/to/environment.p12")
+	if err != nil {
+		panic(err)
+	}
 
-configuration := configuration.New(
-    configuration.TestEnvironment,
-    &configuration.Pkcs12{Content: certificate), Password: "p12 password"},
-)
+	config := configuration.New(
+		configuration.TestEnvironment,
+		&configuration.Pkcs12{Content: certificate, Password: "p12 password"},
+	)
 
-bankId := bankid.New(configuration)
+	bankId := bankid.New(config)
 
-payload := bankid.SignPayload{PersonalNumber: "<INSERT PERSONAL NUMBER>", EndUserIp: "192.168.1.1", UserVisibleData: "Test"}
+	payload := bankid.SignPayload{PersonalNumber: "<INSERT PERSONAL NUMBER>", EndUserIP: "192.168.1.1", UserVisibleData: "Test"}
 
-response, err := bankId.Sign(&payload)
+	ctx := context.Background()
+	response, err := bankId.Sign(ctx, &payload)
 
-if err != nil {
-    if response := bankid.UnwrapErrorResponse(err); response != nil {
```

Comment:
didn't find the `UnwrapErrorResponse()` function, but it works fine if try to cast it (tested)